il.co.ravkavonline.RavKavOnline.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: il.co.ravkavonline.RavKavOnline
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: ravkavonline
+finish-args:
+  - --persist=.config/ravkavonline
+  - --persist=.local/share/ravkavonline
+  - --share=network
+  - --socket=pcsc
```

Comment:
No x11 or wayland?

Reply:
There's no need for these as there's no GUI.  
The app is started by the web browser when the web-app/website opens a URI with mime type `x-scheme-handler/ravkav`.  
When it runs, the app connects a backend service of the web-app to the PCSC daemon.  
The process is kept alive while the user still has the web-app session active in the web browser.